zulip_botserver/zulip_botserver/server.py

```diff
@@ -178,8 +212,16 @@ def handle_bot() -> str:
 def main() -> None:
     options = parse_args()
     global bots_config
+
     try:
-        bots_config = read_config_file(options.config_file, options.bot_name)
+        if options.use_env_vars:
+            bots_config = read_config_from_env_vars(options.bot_name)
+        elif options.config_file:
+            bots_config = read_config_file(options.config_file, options.bot_name)
+        else:
+            sys.exit("""Configuration Error: Please pass --use-env-vars or -ev along with properly formatted environment varibales
```

Comment:
```suggestion
            sys.exit("""Configuration Error: Please pass --use-env-vars or -e along with properly formatted environment varibales
```

Reply:
Thanks for this probably I was a bit overexcited for my 1st PR at Zulip